README.md

```diff
@@ -1,22 +1,19 @@
-# Ruby Project Template
-
-1 - On Github, on the main page of the Gameportal repository, click the Code button and copy "Clone with HTTPS" by clicking the copy icon.
-In your terminal, enter a git clone and past the link that you copied from the repository, and press the   Enter
-(e.g. git clone https://github.com/shadabwahidullah/Ruby-project-template.git).
+# Test Driven Project
 
+In this project, we used the Test Driven approach by doing first the tests and then writing the code that successfully passes all the tests
 - To get a local copy up and running follow these steps.
 
 2 - open your terminal and clone the project running 
 
-`https://github.com/shadabwahidullah/Ruby-project-template.git`
+`https://github.com/shadabwahidullah/Test-Driven-Project.git`
 
 3 - install all the gems running
 
 `bundle install`
 
 4 - to run the file  use
 
-`Ruby morse.rb`
+`Ruby solver.rb`
```

Comment:
it can be good to add testing instructions as well, currently, it's not clear as to which commands to run when you want to test.